spec/features/patients/index_spec.rb

```diff
@@ -52,7 +52,7 @@
 
     it "landing page - adherence follow up patient card detail- with bp" do
       chc_bagta_facility = create(:facility, facility_group: ihmi_facility_group, name: "bagta")
-      var_patient=create(:patient ,registration_facility: chc_bagta_facility, device_created_at: 2.day.ago)
+      var_patient=create(:patient, age: 50, registration_facility: chc_bagta_facility, device_created_at: 2.day.ago)
```

Comment:
@harimohanraj89 We do not have a way to check code-style issues at the moment. We use standard RubyMine code-style configurations (and occasionally ignore it in special cases). 

Code style issues in PR comments are and should be pointed out, I personally tend to prefix a `[STYLE]` in the comment to distinguish it from other comments.

Just a side-note however, some older Feature Specs from Prerna don't follow the correct code-style since they were written outside of RubyMine and should be corrected as we work on them.

Reply:
Awesome 👍 Would you be open to trying some automated tooling to take care of the trivial style issues? It could save a lot of effort during PR review. eg. Codeclimate + Rubocop